packages/fiori/src/IllustratedMessage.hbs

```diff
@@ -6,9 +6,9 @@
 		<div class="ui5-illustrated-message-text-and-actions-container">
 			{{#if hasTitle}}
 					{{#if hasFormattedTitle}}
-						<slot name="title"></slot>
+						<slot name="title" class="ui5-illustrated-message-title-slot"></slot>

```

Comment:
Looks good. We might as well implement this like the subtitle. Wrap both the `slot` and the internal `ui5-title` templates in an additional div and give it the class `ui5-illustrated-message-title`

```hbs
<div part="subtitle" class="ui5-illustrated-message-title">
	{{#if hasFormattedTitle}}
		<slot name="title"></slot>
	{{else}}
		<ui5-title level="{{titleLevel}}" class="ui5-illustrated-message-title" wrapping-type="Normal">{{effectiveTitleText}}</ui5-title>
	{{/if}}
</div>
```

That way we will maintain a single class and a consistent structure with the `subtitle` DOM.